common/src/app/common/geom/shapes/transforms.cljc

```diff
@@ -342,6 +344,10 @@
 ;;   tells if the resize vectors must be applied to text shapes
 ;;   or not.
 
+(defn empty-modifiers? [modifiers]
+  (or (nil? modifiers)
+      (empty? (d/without-keys modifiers [:ignore-geometry?]))))
```

Comment:
I think there the without-keys is not necessary, `(empty? (dissoc modifiers :ignore-geometry))` should do the work much efficiently.

Reply:
You're right. I'll change it